recipes/ruby/build.sh

```diff
@@ -0,0 +1,9 @@
+set -e
```

Comment:
Would add `#!/bin/bash` at the top of the file. `conda-build` does the right thing mostly. So this is more of a style thing and an extra hint to editors.